### PR TITLE
fix(depots): depots deleted by name

### LINKED ISSUE #2110: bug: Flaky Tests in Stock entry
The Stock Entry module still seems to have a flaky tests in `master`.  See https://travis-ci.org/IMA-WorldHealth/bhima-2.X/builds/277753401 as an example of this.

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the depots test to ensure that it only removes depots
by the correct name.  This is thought to contribute to #2110.

~~Closes #2110.~~

----

Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through eslint.
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the online review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda @DedrickEnc , this should solve all the breaking tests in our PRs.  If one of you gets a chance, it would be helpful to review it as soon as possible. :+1:

File: test/end-to-end/depots/depots.page.js [hunk 1]
```diff
@@ -70,10 +69,14 @@ class DepotPage {
...
-    components.modalAction.confirm();
-    components.notification.hasSuccess();
+  deleteDepot(text) {
+    GU.getGridIndexesMatchingText(this.gridId, text)
+      .then(indices => {
+        const { rowIndex } = indices;
+        GA.clickOnMethod(rowIndex, this.actionLinkColumn, 'delete', this.gridId);
+        components.modalAction.confirm();
+        components.notification.hasSuccess();
+      });
```
Comment (mbayopanda) [inline test/end-to-end/depots/depots.page.js:83]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM. E2E and Integration tests pass without any problems

**Comment by jniles** [comment]
Hmm... this didn't fix it.  [Build Failure](https://travis-ci.org/IMA-WorldHealth/bhima-2.X/jobs/278597401).

**Comment by jniles** [comment]
Let's try replacing that `edit` action with the `getGridIndexesMatchingText()` and see if that fixes it.

**Comment by jniles** [comment]
Hey, this is a new error! :tada:

**Comment by jniles** [comment]
Alright, let's try this again.

bors r+

**Pull request merged**